modules/snooker/src/main/ts/ephox/snooker/api/CopySelected.ts

```diff
@@ -79,46 +89,78 @@ const fillInGaps = <T>(list: RowData<T>[], house: Warehouse, stats: StatsStruct,
   }
 };
 
-const clean = (table: SugarElement, stats: StatsStruct): void => {
+const clean = (replica: SugarElement<HTMLTableElement>, stats: StatsStruct, widthDelta: number): void => {
   // can't use :empty selector as that will not include TRs made up of whitespace
-  const emptyRows = Arr.filter(LayerSelector.firstLayer(table, 'tr'), (row) =>
+  const emptyRows = Arr.filter(LayerSelector.firstLayer(replica, 'tr'), (row) =>
     // there is no sugar method for this, and Traverse.children() does too much processing
     (row.dom as HTMLElement).childElementCount === 0
   );
   Arr.each(emptyRows, Remove.remove);
 
   // If there is only one column, or only one row, delete all the colspan/rowspan
   if (stats.minCol === stats.maxCol || stats.minRow === stats.maxRow) {
-    Arr.each(LayerSelector.firstLayer(table, 'th,td'), (cell) => {
+    Arr.each(LayerSelector.firstLayer(replica, 'th,td'), (cell) => {
       Attribute.remove(cell, 'rowspan');
       Attribute.remove(cell, 'colspan');
     });
   }
 
-  Attribute.remove(table, 'width');
```

Comment:
The original issue for this JIRA was that these attributes were being removed, so should there be a "Fixed" changelog entry for that as well?